### PR TITLE
bare minimum update work for ember 2

### PR DESCRIPTION
**Pull request by mkorfmann**

### CODE REVIEW

**Comment by mkorfmann** [comment]
Hey @hhff,

My suggestion is that this PR is merged first, than I'll rebase #69 in order to get it merged as well.

**Comment by mkorfmann** [comment]
Or maybe #69 won't need additional rebasing, can't think of any conflicting changes right now.

**Comment by mkorfmann** [comment]
Let's see...

**Comment by hhff** [comment]
Looks good to me.  Man i really gotta figure out how to make vim strip whitespace...

@mike-north - want to get your thoughts on the Ember Try stuff here.  Also @kellyselden could u have a peeper

:heart: this is a big one!  gonna be our first minor release....!

File: config/ember-try.js [hunk 1]
```diff
@@ -39,6 +39,16 @@ module.exports = {
       }
     },
     {
+      name: 'ember-release-with-proper-ember-data',
+      dependencies: {
+        'ember': 'components/ember#release',
+        'ember-data': '2.0.0-beta.2'
```
Comment (kellyselden) [inline config/ember-try.js]: ember data 2.0 is no longer in beta

Comment (mkorfmann) [inline config/ember-try.js]: thanks! just fixed it :+1:

**Comment by mkorfmann** [comment]
Hey @kellyselden, can you explain why ember-data is needed as a npm package and bower component, both?

**Comment by mkorfmann** [comment]
Hey @hhff, @kellyselden, @mike-north:

Is there still something to do here apart from squashing the commits?

**Comment by hhff** [comment]
I'll defer to @mike-north on that, he's the resident ember-try expert

**Comment by hhff** [comment]
friendliest of friendly pings @mike-north

File: .travis.yml [hunk 2]
```diff
@@ -15,15 +15,12 @@ env:
   - EMBER_TRY_SCENARIO=ember-1.12
   - EMBER_TRY_SCENARIO=ember-1.13
   - EMBER_TRY_SCENARIO=ember-release
+  - EMBER_TRY_SCENARIO=ember-release-ember-data-1.13.1x
   - EMBER_TRY_SCENARIO=ember-beta
   - EMBER_TRY_SCENARIO=ember-canary
 
 matrix:
   fast_finish: true
```
Comment (mike-north) [inline .travis.yml]: there's no point in `fast_finish` if you have no `allowed_failures`

File: bower.json [hunk 3]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "ember-infinity",
   "dependencies": {
-    "ember": "1.13.7",
+    "ember": "1.13.9",
+    "ember-data": "1.13.9",
```
Comment (mike-north) [inline bower.json]: I would keep these in alphabetical order, so that subsequent upgrades don't churn bower.json excessively

File: package.json [hunk 4]
```diff
@@ -18,6 +18,7 @@
   "author": "Hugh Francis",
   "license": "MIT",
   "devDependencies": {
+    "ember-data": "1.13.11",
```
Comment (mike-north) [inline package.json]: may not be necessary

**Comment by mike-north** [comment]
:+1: once fixes are made

File: package.json (hunk 4 above)
Comment (mkorfmann) [inline package.json]: Totally true! I don't know how ember-data ended up in the package.json, somehow I thought it was there before up until this point. Even asked @kellyselden why this is needed... :)

Comment (mkorfmann) [inline package.json]: Hey! I thought so too at first, but I tried removing it with no luck. Almost all tests are failing when ember-data is removed from package.json.

Comment (mkorfmann) [inline package.json]: I was hoping someone could enlighten me on this...

File: bower.json (hunk 3 above)
Comment (mkorfmann) [inline bower.json]: Oh, thanks! Didn't notice the pattern there.

**Comment by mkorfmann** [comment]
Thanks @mike-north!

@hhff:

Should be ready to go now! :)

File: package.json (hunk 4 above)
Comment (mike-north) [inline package.json]: leave it in then. I'll take a look from the ember-cli side at some point

**Comment by hhff** [comment]
YES!  Nice one dudes!

**Pull request merged**